docs/source/index.rst

```diff
@@ -36,6 +36,7 @@ pyro: a python hydro code
    multigrid
    diffusion_basics
    incompressible_basics
+   incompressible_viscous_basics
```

Comment:
That might make the page a little cumbersome.  An alternative is to combine the solvers into a single one, since incompressible_viscous perfectly reproduces incompressible when viscosity=0.  But that would be another PR.

Thoughts @zhichen3  @zingale ?